packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -7,42 +7,56 @@
  * @prop --calcite-tile-border-color: Specifies the border color of the component.
  * @prop --calcite-tile-description-text-color: Specifies the description text color of the component.
  * @prop --calcite-tile-heading-text-color: Specifies the heading text color of the component.
+ * @prop --calcite-tile-icon-color: Specifies the color of the selected icon in the component.
+ * @prop --calcite-tile-selection-icon-color-hover: Specifies the color of the selection icon in the component on hover.
  */
 
+/** Internal variables
+  * --calcite-internal-tile-border-color
+  * --calcite-internal-tile-description-text-color
+  * --calcite-internal-tile-heading-text-color
+  * --calcite-internal-tile-icon-color
+  * --calcite-internal-tile-outline-color
+  * --calcite-internal-tile-selection-icon-color
+  */
+
 :host {
-  --calcite-tile-background-color: var(--calcite-color-foreground-1);
-  --calcite-tile-border-color: var(--calcite-color-border-2);
-  --calcite-tile-description-text-color: var(--calcite-color-text-3);
-  --calcite-tile-heading-text-color: var(--calcite-color-text-2);
-  --calcite-ui-icon-color: var(--calcite-color-text-3);
+  --calcite-internal-tile-border-color: var(--calcite-color-border-2);
+  --calcite-internal-tile-description-text-color: var(--calcite-color-text-3);
+  --calcite-internal-tile-heading-text-color: var(--calcite-color-text-2);
 
   box-sizing: border-box;
   display: inline-block;
 }
 
 .container {
-  background-color: var(--calcite-tile-background-color);
+  --calcite-internal-tile-outline-color: var(--calcite-tile-border-color, var(--calcite-internal-tile-border-color));
+  --calcite-internal-tile-select-icon-color: var(--calcite-tile-selection-icon-color, var(--calcite-color-text-3));
+
+  background-color: var(--calcite-tile-background-color, var(--calcite-color-foreground-1));
   block-size: var(--calcite-container-size-content-fluid);
   box-sizing: border-box;
   inline-size: var(--calcite-container-size-content-fluid);
-  outline: var(--calcite-border-width-sm, 1px) solid var(--calcite-tile-border-color);
+  outline: var(--calcite-border-width-sm, 1px) solid var(--calcite-internal-tile-outline-color);
   user-select: none;
-  &.interactive {
+  box-shadow: var(--calcite-tile-shadow, var(--calcite-internal-tile-shadow, none)) &.interactive {
```

Comment:
Syntax error here.  Also, `--calcite-tile-shadow` is a vague name and also not documented above.  I'm wondering if we even need to expose a css variable for this or not since I imagine a user will want to customize the `brand` color and have it apply globally to all components?  And, if they need to override the `brand` color specific to Tiles, they should be able to do `calcite-tile { --calcite-color-brand: #custom; }`, right?

Reply:
Just tested this css and it works great:
```
.screenshot-test calcite-tile {
  --calcite-color-brand: green;
}
```

<img width="559" alt="image" src="https://github.com/Esri/calcite-design-system/assets/821864/964810d1-1719-494a-842d-88312e65883f">